### ISSUE TITLE
Не удается установить зависимости

### ISSUE THREAD

**Issue by danilaeremin** [YandexLyceumPP/Resume#3]
ERROR: Command errored out with exit status 1:
   command: /opt/homebrew/opt/python@3.9/bin/python3.9 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/76/54lfwr6d0cxbc7zsctsvpf5mwrmkdv/T/pip-install-wpy3hxpi/psycopg2_8676bc6d53ea43b5a13b06b3bb3f6d2a/setup.py'"'"'; __file__='"'"'/private/var/folders/76/54lfwr6d0cxbc7zsctsvpf5mwrmkdv/T/pip-install-wpy3hxpi/psycopg2_8676bc6d53ea43b5a13b06b3bb3f6d2a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/76/54lfwr6d0cxbc7zsctsvpf5mwrmkdv/T/pip-pip-egg-info-xml4n4ti
       cwd: /private/var/folders/76/54lfwr6d0cxbc7zsctsvpf5mwrmkdv/T/pip-install-wpy3hxpi/psycopg2_8676bc6d53ea43b5a13b06b3bb3f6d2a/
  Complete output (20 lines):
  running egg_info
  creating /private/var/folders/76/54lfwr6d0cxbc7zsctsvpf5mwrmkdv/T/pip-pip-egg-info-xml4n4ti/psycopg2.egg-info
  writing manifest file '/private/var/folders/76/54lfwr6d0cxbc7zsctsvpf5mwrmkdv/T/pip-pip-egg-info-xml4n4ti/psycopg2.egg-info/SOURCES.txt'
  
  Error: pg_config executable not found.
  
  pg_config is required to build psycopg2 from source.  Please add the directory
  containing pg_config to the $PATH or specify the full executable path with the
  option:
  
      python setup.py build_ext --pg-config /path/to/pg_config build ...
  
  or with the pg_config option in 'setup.cfg'.
  
  If you prefer to avoid building psycopg2 from source, please install the PyPI
  'psycopg2-binary' package instead.
  
  For further information please check the 'doc/src/install.rst' file (also at
  <https://www.psycopg.org/docs/install.html>).

**Comment by rodion-gudz** [comment]
@danilaeremin  к сожалению, это проблема вашей системы... Судя по ошибке, у вас тоже машинка на apple silicon, сам лично с таким сталкивался, от нас увы не зависит 😌

**Comment by rodion-gudz** [comment]
Ну а суть ошибки в том, что под apple silicon видимо нет билда нужных зависимостей для этой библиотеки (psycopg2), которые на ходу она собрать не может

**Comment by rodion-gudz** [comment]
@danilaeremin можете попробовать `pip install psycopg2-binary`, вероятно поможет

**Comment by danilaeremin** [comment]
Да, у меня на M1 компьютер и билды существуют, помогает аккуратно писать зависимости, как правило не через просто .txt, а через setup.py, который честно смотрит систему и генерирует их
